DESCRIPTION

```diff
@@ -19,3 +19,5 @@ Imports:
     magrittr,
     tidyr,
     stringr
+Suggests: 
+    testthat
```

Comment:
Good look on adding testthat to the `Suggests` I didn't think you have to do that. Thanks to https://github.com/r-lib/testthat/issues/103 I learned otherwise!

Reply:
I probably used  `usethis::use_testthat()`, which sets things up + adds to `Suggests`.